lib/ipinfo-rails.rb

```diff
@@ -2,18 +2,25 @@
 
 require 'rack'
 require 'ipinfo'
+require 'ipinfo-rails/ipselector/default_ipselector'
 
 class IPinfoMiddleware
     def initialize(app, cache_options = {})
```

Comment:
While we're here, can we rename `cache_options` to just `options` since it's for more than just cache options?